src/replication/state.cpp

```diff
@@ -111,7 +111,7 @@ bool ReplicationState::TryPersistRoleMain(std::string new_epoch, utils::UUID mai
   return false;
 }
 
-bool ReplicationState::TryPersistUnregisterReplica(std::string_view name) {
+bool ReplicationState::TryPersistUnregisterReplica(std::string_view name) const {
```

Comment:
Same thing `durability_->Delete(key)` is modifying a resource. Should not be considered `const`

Reply:
done